projects/canopy/src/lib/card/card-hero-img/card-hero-img.component.html

```diff
@@ -0,0 +1,12 @@
+<div
+  *ngIf="imageUrl"
+  class="lg-card-hero-image__content"
+  [style.background-image]="'url(' + imageUrl + ')'"
+></div>
+<img
+  *ngIf="imageSrc"
+  alt="card-hero-img"
+  [attr.src]="imageSrc"
```

Comment:
nit: why attr.src instead of just src?

Reply:
I think this was left from a copy paste mistake. Changed it to just 'src' as it should be. :)